src/libs/vtools/dialogs/tools/dialogmove.cpp

```diff
@@ -484,7 +486,7 @@ void DialogMove::originChanged(const QString &text)
 {
     VisToolMove *operation = qobject_cast<VisToolMove *>(vis);
     SCASSERT(operation != nullptr)
-    if (text == "Center Point")
+    if (text == tr("Center Point"))
```

Comment:
One question, because I really dont know: Here its with capital P, while in the vmovetool with lowercase p for point. Does that matter?

Reply:
Yes.